locale/en/blog/release/v10.0.0.md

```diff
@@ -138,7 +138,7 @@ The following APIs have been deprecated in Node.js 10.0.0
 * [[`876836b135`](https://github.com/nodejs/node/commit/876836b135)] - **(SEMVER-MAJOR)** **benchmark**: rename file (Ruben Bridgewater) [#18790](https://github.com/nodejs/node/pull/18790)
 * [[`e9ec9ff269`](https://github.com/nodejs/node/commit/e9ec9ff269)] - **(SEMVER-MAJOR)** **benchmark**: add buffer fill benchmark (Ruben Bridgewater) [#18790](https://github.com/nodejs/node/pull/18790)
 * [[`94d64877ff`](https://github.com/nodejs/node/commit/94d64877ff)] - **(SEMVER-MAJOR)** **benchmark**: improve buffer.readInt(B|L)E benchmarks (Rich Trott) [#11146](https://github.com/nodejs/node/pull/11146)
-* [[`9d4ab90117`](https://github.com/nodejs/node/commit/9d4ab90117)] - **(SEMVER-MAJOR)** **buffer**: do deprecation warning outside `node\_modules` (Anna Henningsen) [#19524](https://github.com/nodejs/node/pull/19524)
```

Comment:
same here ^
I think backslash is required.

Reply:
The same:

![bb2](https://user-images.githubusercontent.com/10393198/39218043-3f56a1de-482b-11e8-9e57-5ba9692742a2.png)